app/src/main/res/values-en-rCA/strings.xml

```diff
@@ -12,7 +12,7 @@
   <string name="menu_open_with_default_browser2">Open in %1$s</string>
   <string name="menu_open_with_a_browser2">Open in…</string>
   <string name="menu_add_to_home_screen">Add to Home screen</string>
-  <string name="menu_whats_new">What\'s New</string>
+  <string name="menu_whats_new">What’s New</string>
```

Comment:
This change looks incorrect. Not escaping the single quote character seems broken.

Reply:
@MickeyMoz is a bot btw. :D

Afaik this doesn't need escaping because it's a different quote ("right single quotation mark" vs. "apostrophe").